app/build.gradle.kts

```diff
@@ -0,0 +1,220 @@
+            isShrinkResources = true
+            isDebuggable = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+
+    buildFeatures {
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.2.0"
+    }
+
+    packagingOptions {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+
+    flavorDimensions("version")
+    productFlavors {
+        create("prod") {
+            dimension = "version"
+            applicationId = "com.kks.nimblesurveyjetpackcompose"
+            buildConfigField("String", "BASE_URL", "\"https://survey-api.nimblehq.co/\"")
+        }
+
+        create("dev") {
+            dimension = "version"
+            // Staging AppId should be different from the real production.
+            applicationId = "com.kks.nimblesurveyjetpackcomposestaging"
+            buildConfigField("String", "BASE_URL", "\"https://survey-api.nimblehq.co/\"")
+        }
+    }
+
+
+    signingConfigs {
+        //keytool -genkey -v -keystore debug.keystore -alias debug-key-alias -keyalg RSA -keysize 2048 -validity 10000
+        getByName("debug") {
+            storeFile = file("../config/debug.keystore")
+            storePassword = "123456"
+            keyAlias = "debug-key-alias"
+            keyPassword = "123456"
+        }
+        create("release") {
+            try {
+                storeFile = file("../config/nimblesurveycompose.keystore")
+                storePassword = props.getProperty("KEYSTORE_PASSWORD") as String
+                keyAlias = props.getProperty("KEY_ALIAS") as String
+                keyPassword = props.getProperty("KEY_PASSWORD") as String
+            } catch (ex: Exception) {
+                throw InvalidUserDataException("You should define KEYSTORE_PASSWORD and KEY_PASSWORD in gradle.properties.")
+            }
+        }
+    }
+
+    applicationVariants.all {
+        kotlin.sourceSets {
+            getByName(name) {
+                kotlin.srcDir("build/generated/ksp/$name/kotlin")
+            }
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    kotlinOptions.jvmTarget = "1.8"
+
+    lintOptions {
+        isAbortOnError = false
+    }
+
+    configurations {
+        androidTestImplementation {
+            exclude(group = "io.mockk", module = "mockk-agent-jvm")
+        }
+    }
+
+    kapt {
+        arguments {
+            arg("room.schemaLocation", "$projectDir/schemas")
+        }
+    }
+
+    testOptions {
+        animationsDisabled = true
+    }
+}
+
+dependencies {
+
+    implementation("androidx.core:core-ktx:1.7.0")
```

Comment:
Shall we extract this version and all the versions below to `Versions`? What do you think? 😄